compiler/src/dotty/tools/dotc/core/classfile/ClassfileParser.scala

```diff
@@ -1058,13 +1059,18 @@ class ClassfileParser(
    */
   private def enterOwnInnerClasses()(using Context, DataReader): Unit = {
     def enterClassAndModule(entry: InnerClassEntry, file: AbstractFile, jflags: Int) =
-      SymbolLoaders.enterClassAndModule(
+      val (cls, mod) = SymbolLoaders.enterClassAndModule(
           getOwner(jflags),
-          entry.originalName,
+        entry.originalName,
           new ClassfileLoader(file),
           classTranslation.flags(jflags),
           getScope(jflags))
 
+      val privateWithin = getPrivateWithin(jflags)
```

Comment:
`if privateWithin.exists then` here as well for consistency?

Reply:
Sure